react-graphql-apollo-tutorial.md

```diff
@@ -878,7 +882,7 @@ const ErrorMessage = ({ error }) => (
 export default ErrorMessage;
 {{< /highlight >}}
 
-Try to change the naming of a field in your query to something that is not offered by GitHub's GraphQL API and see what's rendered in the browser. You should see something like this: *Error: GraphQL error: Field 'viewers' doesn't exist on type 'Query'*. Or if you simulate being offline you should get something similar to this: *Error: Network error: Failed to fetch*. That's how errors can be differentiated into GraphQL errors and network errors. In conclusion, that's how you can handle errors on a component (or query) level when doing queries but also mutations later on. So how could this error handling be implemented on an application level? First, there is another Apollo package which can be installed for this use case:
+Try to change the name of a field in your query to something not offered by GitHub's GraphQL API, and observe what's rendered in the browser. You should see something like this: *Error: GraphQL error: Field 'viewers' doesn't exist on type 'Query'*. Or, if you simulate offline functionality, you'll see: *Error: Network error: Failed to fetch*. That's how errors can be separated into GraphQL errors and network errors. You can handle errors on a component or query level but it will also help with mutations latere. To implement this into an application, install another Apollo package:
```

Comment:
"You can handle errors on a component or query level but it will also help with mutations latere" to "You can handle errors on a component or query level but it will also help with mutations later"

Reply:
"To implement this into an application, install another Apollo package:" -> "To implement error handling on an application level, install another Apollo package:"